src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -86,9 +86,9 @@ trait OuroborosConfigBuilder {
   lazy val ouroborosConfig = OuroborosConfig(Config.config)
 }
 
-trait ElectionManagerBuilder {
-  self: OuroborosConfigBuilder =>
-  lazy val electionManager = ElectionManagerImpl(ouroborosConfig)
+trait CertificateAuthorityManagerBuilder {
+  self: LedgerBuilder with BlockchainBuilder with OuroborosConfigBuilder with BlockchainConfigBuilder =>
```

Comment:
Minor detail: Is `LedgerBuilder` needed here?

Reply:
:+1: